### PR TITLE
Feature/ATG-13 State-controls Component

### PR DESCRIPTION
**Pull request by IraCherginyshka**
Create html template with styles state-controls-component

### CODE REVIEW

File: src/app/components/game-state-controls/game-state-controls.component.html [hunk 1]
```diff
@@ -1 +1,8 @@
-<p>game-state-controls works!</p>
+<div class="state-controls">
+  <button class="state-control rotation">rotation</button>
+  <div class="horizontal">
+    <button class="state-control right">right</button>
+    <button class="state-control left">left</button>
```
Comment (oleksandrstarov) [inline src/app/components/game-state-controls/game-state-controls.component.html]: I would swap these control names - as on the screen we have right label on the left side... Looks confusing

File: src/app/components/game-state-controls/game-state-controls.component.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    background-color: $background-color;
+    border: $hover-border-controls $secondary-color;
+    border-radius: 10px;
+    text-align: center;
+    font-size: inherit;
+    text-transform: inherit;
+    font-family: inherit;
+    color: $default-color;
+
+    &:hover {
```
Comment (oleksandrstarov) [inline src/app/components/game-state-controls/game-state-controls.component.scss:24]: I would expect cursor: pointer; on hover for all clickable controls on application

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: src/app/components/game-state-controls/game-state-controls.component.html (hunk 1 above)
Comment (IraCherginyshka) [reply src/app/components/game-state-controls/game-state-controls.component.html]: You're right!
Done

File: src/app/components/game-state-controls/game-state-controls.component.scss (hunk 2 above)
Comment (IraCherginyshka) [reply src/app/components/game-state-controls/game-state-controls.component.scss:24]: Done!

**Review by oleksandrstarov — APPROVED** [review]

**Pull request merged**